src/javascript/app/base/footer.js

```diff
@@ -27,6 +28,42 @@ const Footer = (() => {
         const $status_notification = $('#status_notification');
         $status_notification.slideUp(200);
     };
+
+    const adjustElevioAndScrollup = (elevio_height, scrollup_height) => {
+        const $elevio_button = $('#_elev_io ._6byvm');
+        const $scrollup = $('#scrollup');
+        $elevio_button.attr('style', `bottom: ${elevio_height}px !important`);
+        $scrollup.attr('style', `bottom: ${$elevio_button.height() + scrollup_height}px`);
```

Comment:
it's better if we can add classes to the elements that have these styles instead of changing css in javascript